apps/mull-ui/src/app/pages/home/discover/discover-page.tsx

```diff
@@ -22,7 +22,11 @@ export const DiscoverPage = ({ history }) => {
   }
   return (
     <div className="discover-page-tabs-container" data-testid="discover-tab">
-      {eventCards}
+      {eventCards && eventCards.length !== 0 ? (
```

Comment:
Slightly easier to read, if you agree we could apply this everywhere else
```suggestion
      {eventCards && eventCards.length > 0 ? (
```